src/modules/Sharpen/Sharpen.js

```diff
@@ -0,0 +1,44 @@
+// Generates a 3x3 convolution sharpening kernel
+function kernelGenerator(strength = 1) { //default value of strength set to 1
+
+  let kernel = [[0, -1 * strength, 0], [-1 * strength, 5 * strength, -1 * strength], [0, -1 * strength, 0]];
```

Comment:
```suggestion
  let kernel = [
    [0, -1 * strength, 0],
    [-1 * strength, 5 * strength, -1 * strength],
    [0, -1 * strength, 0]
  ]
```

Just so that it is more readable :)